src/modules/navigator/rtl_direct.cpp

```diff
@@ -370,6 +380,7 @@ void RtlDirect::set_rtl_item()
 	} else {
 		// Convert mission item to current position setpoint and make it valid.
 		if (mission_item_to_position_setpoint(_mission_item, &pos_sp_triplet->current)) {
+			pos_sp_triplet->current.alt_acceptance_radius = altitude_acceptance_radius;
```

Comment:
Remind me again - with NAN it uses the default altitude acceptance radius?

Reply:
Yes, if it is NAN it uses the default one set from the params.